File: Kickstart/recordday.py
# a = [1,2,0,7,2,0,2,0]

# b = [4,8,15,16,23,42]

# c = [3,1,4,1,5,9,2,6,5]

# d = [9,9,9,9,9,9]


def checkrecord(i):
    count = 0

    #first day
    if(i[0] > i[1]):
        count +=1
    
    #for all other days
    for k in range(1,len(i)-1):
        prev  = max(i[:k])
        try:
            if(i[k] > prev and i[k] > i[k+1]):
                count+=1
        except:
            pass
    
    #last day
    if(i[-1] > max(i[:-1])):
        count +=1

    return count


testcases = int(input())
for t in range(testcases):
    leng = int(input())
    inputs = list(input().split(" "))
    print("Case #" + str(t+1) + ":" + " " + str(checkrecord(inputs)))

File: Kickstart/recordday.cpp
#include <iostream>
#include <vector>
using namespace std;
#include <algorithm>
 
int calculated(vector<int> i,int nos)
{
    int count = 0 ; 

    // first day
    if(i[0] > i[1])
    {
        ++count ; 
    }
        
    
    // for all other days

    for(int k  = 1; k < nos -1 ; k++)
    {
        int prev  = * max_element(i.begin(),i.begin() + k) ; 

        if(i[k] > prev and i[k] > i[k+1])
        {
            count++ ;
        }
                

    }
    
    // last day
    if(i[nos-1] > * max_element(i.begin(),i.begin()+nos-1))
    {
        count +=1;
    }
        
    return count;
}

int main()
{
    
    int cases;
    cin>>cases;
    int nos = 0 ;
    int value = 0; 

    for(int i = 0  ; i < cases ; i++)
    {
        vector <int> V;
        cin >> nos;
        for(int j = 0 ; j < nos ; j ++ )
        {
            cin >> value;
            V.push_back(value);
            
        }
        
        cout<<"Case #" << i+1 << ": " <<calculated(V,nos) ;  
        if(cases - 1 != i)
            cout<<endl;
    }
}